e2e/playwright/special-permissions-actions-available/src/tests/other-permissions.ts

```diff
@@ -0,0 +1,387 @@
+/*!
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Alfresco Example Content Application
+ *
+ * This file is part of the Alfresco Example Content Application.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail. Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * The Alfresco Example Content Application is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The Alfresco Example Content Application is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * from Hyland Software. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { test } from '@alfresco/playwright-shared';
+import * as testData from '@alfresco/playwright-shared';
+import { expect } from '@playwright/test';
+
+export function collaboratorTests(userCollaborator: string, siteName: string) {
+  test.describe('available actions : ', () => {
+    test.beforeEach(async ({ loginPage }) => {
+      await loginPage.navigate();
+      await loginPage.loginUser({ username: userCollaborator, password: userCollaborator });
+    });
+    test('on File Libraries - [C297647]', async ({ myLibrariesPage }) => {
+      const expectedToolbarPrimary = ['Shared Link Settings', 'Download', 'View', 'View Details', 'More Actions'];
```

Comment:
could you move all these arrays to some reusable consts instead of repeating them for each test?